src/components/EditCharacter.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { Grid, Row, Col, Button, Well } from 'react-bootstrap'
+import { Link, Route } from 'react-router-dom'
+
+import EditAbilityScores from './EditAbilityScores'
+
+export default class EditCharacter extends Component {
+  constructor (props) {
+    super(props)
+
+    this.renderCharacterSummary = this.renderCharacterSummary.bind(this)
+  }
+
+  renderCharacterSummary () {
+    return 'Hello!'
+  }
+
+  componentWillMount () {
+    this.props.updateActiveCharacter(this.props.match.params.uid)
+  }
+
+  render () {
+    const abilityScores = this.props.activeCharacter.hasOwnProperty('abilityScores')
+      ? this.props.activeCharacter.abilityScores : {STR: 8, DEX: 8, CON: 8, INT: 8, WIS: 8, CHA: 8}
```

Comment:
1. muti-line ternaries: let this be the last time we see this in your pull-requests 😄 
2. Again, we have seen this a number of times before. What happens if you decide you want to change the default to `{STR: 7, DEX: 7, CON: 7, INT: 7, WIS: 7, CHA: 7}` ? How many places do you have to change it? The answer should be *one*